.github/workflows/test.yml

```diff
@@ -52,10 +52,10 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         run: yarn --frozen-lockfile
 
-      - name: Set up Ruby 2.6
+      - name: Set up Ruby 2.7
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.6.5
+          ruby-version: 2.7.1
```

Comment:
This change will break builds in Bitrise, their virtual machines use version `2.6.5`
We can migrate to 2.7.1 but we will have to add an script to install it on Bitrise

Reply:
This is script is not being used in Bitrise, but change on the iOS Gemfile will break it I think

Reply:
Then we should update all the versions for it, I'll open a PR with that changes to those lock files, thanks for the heads up I thought we overlooked it